docs/bin/plugin_formatter.py

```diff
@@ -125,12 +125,13 @@ def write_data(text, options, outputname, module):
 #####################################################################################
 
 
-def list_modules(module_dir, depth=0):
+def list_modules(module_dir, depth=0, limit_to_modules="all"):
```

Comment:
`limit_to_modules` is currently supposed to be comma-separated string, so modules with mentioned names should not be a problem.